spec/models/like_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.new(name: 'Something',
+                  photo: 'http://localhost:3000/something.jpg',
+                  bio: 'Something test',
+                  post_counter: 0)
+  post = Post.new(
+    title: 'Something',
+    text: 'Something test',
+    author: user,
+    comments_counter: 0,
+    likes_counter: 0
+  )
+  subject do
+    described_class.new(
+      author: user,
+      post:
+    )
+  end
+  it 'is valid with valid attributes' do
+    expect(subject).to be_valid
+  end
+end
```

Comment:
Nice job so far. However, as part of the project requirements, any custom methods in the models should have a corresponding unit test. In view of this kindly add unit test for the `update_post_like_counter` method in the like model here. :wink: